lnwallet/channel.go

```diff
@@ -1884,7 +1884,13 @@ func NewBreachRetribution(chanState *channeldb.OpenChannel, stateNum uint64,
 	if err != nil {
 		return nil, err
 	}
-	localPkScript, err := input.CommitScriptUnencumbered(keyRing.RemoteKey)
+
+	// Since it is the remote breach we are reconstructing, the output going
+	// to us will be a to-remote script with our local params.
+	localDelay := uint32(chanState.LocalChanCfg.CsvDelay)
+	localScript, err := CommitScriptToRemote(
+		chanState.ChanType, localDelay, keyRing.RemoteKey,
+	)
```

Comment:
Would "our" be more clear in this context?

Reply:
Done: https://github.com/lightningnetwork/lnd/pull/3829/commits/80f4ece42aacb0988820c031cc9d4d0b8a3302a3

Reply:
After recent changes our/their is removed again, but I'm open to using it only here.

Reply:
I think our/their would be more clear, because there is only a single perspective in this function. But up to you given our release cycle.

Reply:
Using our/their: https://github.com/lightningnetwork/lnd/pull/3829/commits/3eecaa3bf5b64b3494301b3c19d2cf7cb53c33b5

Let me know what you think.

Reply:
Looks good